src/Uno.UI/UI/Xaml/FrameworkElement.skia.cs

```diff
@@ -76,6 +78,8 @@ public event TypedEventHandler<FrameworkElement, object> Loading
 		}
 
 		private event RoutedEventHandler _loaded;
+
+		[global::Uno.UI.Xaml.GeneratedWeakEvent]
 		public event RoutedEventHandler Loaded
```

Comment:
I think WinUI's Loaded event isn't weak. So not really sure if we should do it that way